### PR TITLE
Add example of custom controller action

### PR DESCRIPTION
**Pull request by hmic**
crud-view#206

### CODE REVIEW

File: docs/customizing-templates.rst [hunk 1]
```diff
@@ -33,6 +33,22 @@ title for any of the fields by using the ``scaffold.fields`` configuration
...
 
+Adding Controller Actions to utilize Crud Actions
+-------------------------------------------------
+
+It's easy to add an action to a controller that makes use of another
+*CrudView* action. This uses the template provided by the add action.
+
+.. code-block:: php
+
+public function login() {
```
Comment (ADmad) [inline docs/customizing-templates.rst]: Naming the method "register" would be a more realistic example.

Comment (hmic) [reply docs/customizing-templates.rst]: Good catch!

**Comment by hmic** [comment]
Took some time to figure out the real behavior ;-)
The first usecase, with providing the 'edit' view explicitely is the one *not* working by now.
The second, this is why i called it the "default" one, does work at of the current state.

Adding the second try with the $name instead of/ in addition to $this->template fixes it in #206

File: docs/customizing-templates.rst [hunk 2]
```diff
@@ -33,6 +33,35 @@ title for any of the fields by using the ``scaffold.fields`` configuration
...
+
+public function account() {
+    $this->Crud->mapAction('account', [
+        'className' => 'Crud.Edit',
+        'view' => 'edit',
+    ]);
+    return $this->Crud->execute(null, $this->Auth->user('id'));
+  }
+
+By default, it can be overwritten by providing a custom ``register.ctp``.
```
Comment (ADmad) [inline docs/customizing-templates.rst]: Shouldn't it be `account.ctp`?

File: docs/customizing-templates.rst [hunk 3]
```diff
@@ -33,6 +33,35 @@ title for any of the fields by using the ``scaffold.fields`` configuration
...
+
+.. code-block:: php
+
+public function account() {
+    $this->Crud->mapAction('account', [
+        'className' => 'Crud.Edit',
+        'view' => 'edit',
+    ]);
+    return $this->Crud->execute(null, $this->Auth->user('id'));
+  }
```
Comment (ADmad) [inline docs/customizing-templates.rst]: Indentation seems off.

Comment (ADmad) [inline docs/customizing-templates.rst]: Indentation seems off.

File: docs/customizing-templates.rst (hunk 2 above)
Comment (hmic) [reply docs/customizing-templates.rst]: nope. I do want it to render the crudview edit template in the account action with the user id of the currently logged in user. this is a real world example.
And this is the one not working without the patch.

File: docs/customizing-templates.rst (hunk 3 above)
Comment (hmic) [reply docs/customizing-templates.rst]: We'll get there...

File: docs/customizing-templates.rst (hunk 2 above)
Comment (ADmad) [reply docs/customizing-templates.rst]: Ah, this statement is for the code block below it, not above.

Comment (hmic) [reply docs/customizing-templates.rst]: should make that more clear, use : instead of .

File: docs/customizing-templates.rst [hunk 4]
```diff
@@ -33,6 +33,35 @@ title for any of the fields by using the ``scaffold.fields`` configuration
         // The rest of the fields to display here
     ]);
 
+Adding Controller Actions to utilize Crud Actions
+-------------------------------------------------
+
+It's easy to add an action to a controller that makes use of another
+*CrudView* action.
+
+This does use the template provided by the edit action:
```
Comment (ADmad) [inline docs/customizing-templates.rst]: Need to use two ":" in .rst files to get a colon.

Comment (hmic) [reply docs/customizing-templates.rst]: Not quite, but the preview in my repo looks good now, finally.
Thanks!

**Pull request merged**